backend-service/src/app.ts

```diff
@@ -0,0 +1,20 @@
+import express, { Request, Response } from "express";
```

Comment:
I find that importing and exporting types without explicitly stating that they are types makes the code much harder for other people to read so I always make it explicit. This is generally recommended and a lot of `tsconfig.json` configurations enforce this so you don't even need to think about it. 

```suggestion
import express, { type Request, type Response } from "express";
```